docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4797,6 +4898,16 @@ also populates the `status` field in the `MasterArbitrationUpdate` as follows:
 * Non-OK (with `status.code` set to `google.rpc.ALREADY_EXISTS`) when the
   controller is determined to be a slave for a given (`device_id`, `role_id`).
 
+The sender need not specify an `election_id`. If the `election_id` is not
+specified, the sender's `election_id` is considered lower than any
+`election_id`, and the sender will thus never become master. This way, if a
+controller is informed by the consensus layer that it is not supposed to be
```

Comment:
1. Does this section better belong up with the section that describes the details of master-arbitration between controller and P4RT-server?
2. I suggest we omit "consensus layer" (we haven't defined it) and simply say something like: "This way, if a controller that knows it is supposed to be the master may choose not to ..."

Reply:
1. After reading over the document again, this actually seems like a good place.  Let me know if you disagree.
2. Done.

Thanks for the review!